journal/src/main/scala/com/evolutiongaming/kafka/journal/Event.scala

```diff
@@ -26,29 +25,14 @@ object Event {
 
     implicit val payloadCodec: Codec[Option[Payload]] = {
 
-      val errEmpty = Err("")
+      // in the original codec, all subtypes were prefixed with an int32 length, even the None option
```

Comment:
what is `the original codec`?

Reply:
now I am lost - is the `original codec` one before this PR? if so, is the `new one` fully compatible?